### PR TITLE
Add new env2mfile command.

### PR DESCRIPTION
**Pull request by jpakkane**
Attempt number two now that the regex backtracking issue has been fixed.

### CODE REVIEW

File: docs/markdown/snippets/env2cross.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Creating a native file is done in the same way:
+
+    meson env2mfile --native -o current_system.txt
+
+This system will detect if the `_FOR_BUILD` environment variables are
+enabled and then uses them as needed.
+
+With this you should be able to convert any envvar-based cross build
+setup to cross and native files and then use those. Thit means, among
```
Comment (Tachi107) [inline docs/markdown/snippets/env2cross.md:30]:
```suggestion
setup to cross and native files and then use those. This means, among
```

**Comment by jpakkane** [comment]
The macOS failure is due to clock skew.

**Comment by jpakkane** [comment]
The cancelled ones are due to the vs image brownout.

**Comment by eli-schwartz** [comment]
Yay, the CI all passes now, so we can reinstate the code.

**Pull request merged**